src/goto-programs/show_goto_functions_json.cpp

```diff
@@ -55,23 +55,26 @@ void show_goto_functions_jsont::show_goto_functions(
   const goto_functionst &goto_functions)
 {
   json_arrayt json_functions;
-  forall_goto_functions(it, goto_functions)
+  for(const auto &function_entry : goto_functions.function_map)
```

Comment:
I do agree that this is the right thing to do, but I am wondering whether it should be left for a larger scale exercise to be applied across the code base? Yet you could also just squash this into your own commit as you had introduced that change in the first place.

Nit picking: there is a typo in your commit message ("Sawp").